### ISSUE TITLE
Vertical position after ColumnLayout is based on the last column rather than the longest

### ISSUE THREAD

**Issue by bdeterling** [ralfstuckert/pdfbox-layout#31]
Example:

```
document = new Document(40, 50, 40, 60);
document.add(new ColumnLayout(2, 0));
for (int i=0; i < 20; ++i) {
  Paragraph p = new Paragraph();
  p.addText("Column 0 Line " + i, 10, PDType1Font.HELVETICA);
  document.add(p);
}
document.add(ColumnLayout.NEWCOLUMN);
for (int i=0; i < 10; ++i) {
  Paragraph p = new Paragraph();
  p.addText("Column 1 Line " + i, 10, PDType1Font.HELVETICA);
  document.add(p);
}
document.add(new VerticalLayout(), VerticalLayoutHint.LEFT);
Paragraph p = new Paragraph();
p.addText("This should be below the first column", 10, PDType1Font.HELVETICA);
document.add(p);
```

![image](https://user-images.githubusercontent.com/17190/26853867-5a259fe2-4ad9-11e7-8224-70ce8316ef86.png)

If that is intentional, is there a way to reset the position to below the first column? Or is there a workaround other than filling the second column with blank paragraphs?

Thanks!

**Comment by ralfstuckert** [comment]
No, this is not intentional ;-)
Smells like a bug, I will have a look at that hopefully at the weekend

**Comment by ralfstuckert** [comment]
Fixed in 1.0.0
